### DISCUSSION TITLE
Throw Away Your Crash

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#492] [Announcements]
### New feature(s)
* Added `base64` attribute option to XMSBT entry for proper binary data support (@Coolsonickirby)

### Note
* If you go online with added slots, you take full responsibility on whatever happens to your Switch or Nintendo Online account.
* We do not endorse going online with added slots and we are only trying to make it more safe to not risk running into trouble with Nintendo. If you get banned, that's on you. If you get disconnected, that's also on you. Choose carefully.

# Read this
Skyline is not shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.13.0'>Throw Away Your Crash</a>.</em>

**Comment by Sanix-Modding** [comment]
I did everything and its still crashing

**Reply by Sanix-Modding** [reply]
pls help me

**Reply by Sanix-Modding** [reply]
ssbu is still crashing

**Reply by AwesomeKirito** [reply]
Ok for anyone crashing on this version is using an outdated smashline.nro, I used a different one and didn't experience any freezing or crashing while picking characters or playing any matches

**Comment by hollowdebianhd-star** [comment]
Sorry, I don't know if this is already answered, pretty new on this page but:

My SSBU is on 10.1.0, and while I tried to search for a version of ARC that does work with this one, I don't know how to download the zip since they look more like a dev based thing that something to install.

Is there any version compatible?
How can I install it?

Thanks.
